### PR TITLE
Add check for unzip

### PR DESCRIPTION
**Pull request by jshum2479**
This add the unzip command check to the MII image,  the container-registry image already have unzip but there may be a version at one point where the unzip is missing.  unzip was not used until we added wdt version check.

### CODE REVIEW

File: operator/src/main/resources/scripts/introspectDomain.sh [hunk 1]
```diff
@@ -119,19 +119,23 @@ if [ ${DOMAIN_SOURCE_TYPE} == "FromModel" ]; then
...
+      trace SEVERE "DomainSourceType is FromModel, tar is missing in the image. Please use an image with tar installed" && exit 1
+    fi
+    command -v unzip
+    if [ $? -ne 0 ] ; then
+      trace SEVERE "DomainSourceType is FromModel, unzip is missing in the image. Please use an image with unzip installed" && exit 1
     fi
     mkdir -p ${DOMAIN_HOME}
     if [ $? -ne 0 ] ; then
-      trace SEVERE "cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
+      trace SEVERE "DomainSourceType is FromModel, cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
```
Comment (tbarnes-us) [inline operator/src/main/resources/scripts/introspectDomain.sh:134]: LGTM. 

Minor nits for the SEVEREs:  I think it helps to put '' around each keywords like 'tar', 'gzip', 'unzip', and 'FromModel', and end sentences with a period '.'.

File: operator/src/main/resources/scripts/introspectDomain.sh [hunk 2]
```diff
@@ -119,19 +119,23 @@ if [ ${DOMAIN_SOURCE_TYPE} == "FromModel" ]; then
...
     fi
```
Comment (tbarnes-us) [inline operator/src/main/resources/scripts/introspectDomain.sh:135]: Minor nit:  It'd be nice to put a test file in the domain home directory just to verify that it's  accessible - so we can  'fail fast' when hitting a directory access issue.

touch ${DOMAIN_HOME}/testaccess.tmp
if [ $? -ne 0 ]; then
   yadayada Cannot write to domain home directory '${DOMAIN_HOME}' yadayayada
fi
rm -f ${DOMAIN_HOME}/testaccess.tmp

File: operator/src/main/resources/scripts/introspectDomain.sh (hunk 1 above)
Comment (rjeberhard) [reply operator/src/main/resources/scripts/introspectDomain.sh:134]: Let's make these edits and then I'll commit.

File: operator/src/main/resources/scripts/introspectDomain.sh (hunk 2 above)
Comment (jshum2479) [reply operator/src/main/resources/scripts/introspectDomain.sh:135]: done

File: operator/src/main/resources/scripts/introspectDomain.sh (hunk 1 above)
Comment (jshum2479) [reply operator/src/main/resources/scripts/introspectDomain.sh:134]: done

File: operator/src/main/resources/scripts/introspectDomain.sh [hunk 3]
```diff
@@ -119,20 +119,25 @@ if [ ${DOMAIN_SOURCE_TYPE} == "FromModel" ]; then
...
-      trace SEVERE "DomainSourceType is FromModel, unzip is missing in the image. Please use an image with unzip installed" && exit 1
+      trace SEVERE "DomainSourceType is FromModel, 'unzip' is missing in the image. Please use an image with 'unzip' installed" && exit 1
     fi
     mkdir -p ${DOMAIN_HOME}
     if [ $? -ne 0 ] ; then
       trace SEVERE "DomainSourceType is FromModel, cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
     fi
+    touch ${DOMAIN_HOME}/testaccess.tmp
+    if [ $? -ne 0 ]; then
+      trace SEVERE "DomainSourceType is FromModel, cannot write to domain home directory '${DOMAIN_HOME}'" && exit 1
```
Comment (tbarnes-us) [inline operator/src/main/resources/scripts/introspectDomain.sh:138]: Missing period.

File: operator/src/main/resources/scripts/introspectDomain.sh (hunk 1 above)
Comment (tbarnes-us) [reply operator/src/main/resources/scripts/introspectDomain.sh:134]: Minor nits still buzzing around:  Still missing periods at end of each SEVERE sentence.  Would be nice to have quotes around FromModel too.

**Pull request merged**